corehq/apps/app_manager/views/view_generic.py

```diff
@@ -193,6 +193,21 @@ def _make_name(suffix):
                 'default_file_name': _make_name('case_list_menu_item'),
                 'qualifier': 'case_list-menu_item_',
             })
+            if toggles.SYNC_SEARCH_CASE_CLAIM.enabled(domain):
```

Comment:
I thought all new changes were going under the USH-specific flag, USH_CASE_CLAIM_UPDATES, at least temporarily.

Reply:
i might have mixed the flags. I thought of this as being under the generic feature flag that permits case search. Isn't `SYNC_SEARCH_CASE_CLAIM` the one? or you are saying that support for multimedia is just for `USH_CASE_CLAIM_UPDATES`.

Reply:
you are right, this was always `USH_CASE_CLAIM_UPDATES`